homeassistant/helpers/template.py

```diff
@@ -508,13 +517,15 @@ def is_safe_callable(self, obj):
 ENV = TemplateEnvironment()
 ENV.filters['round'] = forgiving_round
 ENV.filters['multiply'] = multiply
+ENV.filters['log'] = logarithm
```

Comment:
As far as I can tell from the jinja2 docs, you don't need to add it to `filters` *and* `globals` dict. Globals should be enough, shouldn't it?